bin/kubectl-crossplane-stack-install

```diff
@@ -41,8 +51,8 @@ KUBEY_STACK_IMAGE_NAME=$( echo "${STACK_IMAGE_NAME}" | tr '/' '-' )
 # by passing arguments
 STACK_NAME="${2:-${KUBEY_STACK_IMAGE_NAME}}"
 
-kubectl crossplane stack generate-install "$@" | kubectl apply -f -
-# Printing out the stack request object from the cluster may be useful
+kubectl crossplane stack generate-install "${CLUSTER_OPT}" "$@" | kubectl apply -f -
```

Comment:
@displague I suspect the option may need to be passed unquoted so that it doesn't become an empty string as the first argument to this command when it's not set. Did you already test the behavior with it set to an empty string?

Reply:
@suskin nice catch.. I considered this but did not expect it to accept that as an empty argument.

I tested this again, unquoted, for both the GCP and the wordpress stack:

```
$  kubectl crossplane stack install crossplane/sample-stack-wordpress:master sample-stack-wordpress
stackinstall.stacks.crossplane.io/sample-stack-wordpress created
apiVersion: stacks.crossplane.io/v1alpha1
kind: StackInstall
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"stacks.crossplane.io/v1alpha1","kind":"StackInstall","metadata":{"annotations":{},"name":"sample-stack-wordpress","namespace":"default"},"spec":{"package":"crossplane/sample-stack-wordpress:master"}}
  creationTimestamp: "2019-09-16T17:14:38Z"
  generation: 1
  name: sample-stack-wordpress
  namespace: default
  resourceVersion: "392682"
  selfLink: /apis/stacks.crossplane.io/v1alpha1/namespaces/default/stackinstalls/sample-stack-wordpress
  uid: aa9a6c4c-b280-4553-925e-db074063b3cb
spec:
  package: crossplane/sample-stack-wordpress:master
status:
  conditionedStatus:
    conditions:
    - lastTransitionTime: "2019-09-16T17:14:38Z"
      reason: Managed resource is being created
      status: "False"
      type: Ready
    - lastTransitionTime: "2019-09-16T17:14:38Z"
      reason: Successfully reconciled managed resource
      status: "True"
      type: Synced
  installJob:
    name: sample-stack-wordpress
    namespace: default
```

Reply:
The command succeeded without the quotes. The actual stackinstall job is failing, but I don't believe that reflects on this PR.

Reply:
```
NAMESPACE           NAME                                         READY   STATUS                  RESTARTS   AGE
default             sample-stack-wordpress-wzm8b                 0/1     Init:ImagePullBackOff   0          5m16s
```